ISSUE_SORTING.md

```diff
@@ -1,91 +1,89 @@
 
-### Who triages?
+### Who does the sorting?
 
-Triaging engineers are a `conda` governance [sub-team](https://github.com/conda-incubator/governance#sub-teams); they are a group of Anaconda and community members who are responsible for making decisions regarding closing issues and setting feature work priorities, amongst other triage-related tasks.
+Sorting engineers are a `conda` governance [sub-team](https://github.com/conda-incubator/governance#sub-teams); they are a group of Anaconda and community members who are responsible for making decisions regarding closing issues and setting feature work priorities, amongst other sorting-related tasks.
 
 
-### How do items show up in triaging?
+### How do items show up for sorting?
 
 New issues that are opened in any of the repositories in the [`conda` GitHub project](https://github.com/conda) will show up in the `Sorting` view of the [Planning project](https://github.com/orgs/conda/projects/2/views/11). This process is executed via [GitHub Actions](https://docs.github.com/en/actions). The two main GitHub Actions workflows utilized for this purpose are [Issues](https://github.com/conda/infra/blob/main/.github/workflows/issues.yml) and [Project](https://github.com/conda/infra/blob/main/.github/workflows/project.yml).
 
 The GitHub Actions in the `conda/infra` repository are viewed as canonical; the [Sync workflow](https://github.com/conda/infra/blob/main/.github/workflows/sync.yml) sends out any modifications to [other `conda` repositories](https://github.com/conda/infra/blob/main/.github/sync.yml#L3-L17) from there.
 
 
-### What is done about the issues in "triaging" mode?
+### What is done about the issues in "sorting" mode?
 
-Issues in the "Triaging" column of the board have been reviewed by a triaging engineer and are considered ready for the following procedures:
+Issues in the ["Sorting" tab of the project board](https://github.com/orgs/conda/projects/2/views/11) have been reviewed by a sorting engineer and are considered ready for the following procedures:
 
 - Mitigation via short-term workarounds and fixes
 - Redirection to the correct project
 - Determining if support can be provided for errors and questions
 - Closing out of any duplicate/off-topic issues
 
-The triaging engineers on rotation are not seeking to _resolve_ issues that arise. Instead, the goal is to understand the ticket and to understand whether it is an issue in the first place and to then collect as much relevant information as possible so that the maintainers of `conda` can make an informed decision on the appropriate resolution schedule necessary.
+The sorting engineers on rotation are not seeking to _resolve_ issues that arise. Instead, the goal is to understand the ticket and to determine whether it is an issue in the first place and to then collect as much relevant information as possible so that the maintainers of `conda` can make an informed decision about the appropriate resolution schedule.
```

Comment:
```suggestion
The sorting engineers on rotation are not seeking to _resolve_ issues that arise. Instead, the goal is to understand the ticket and to determine whether it is an issue in the first place, and then to collect as much relevant information as possible so that the maintainers of `conda` can make an informed decision about the appropriate resolution schedule.
```